### PR TITLE
First round of tests for ONNX Conv op.

### PR DESCRIPTION
**Pull request by spandantiwari**
This is a first set of tests for Conv op, which covers some basic functionality. Conv op has many attributes and requires many more test cases, which I plan to add in bite-sized chunks in following PRs.

### CODE REVIEW

File: docs/Operators.md [hunk 1]
```diff
@@ -1376,11 +1473,10 @@ for i in range(len(shape)):
 node = onnx.helper.make_node(
     'Flatten',
     inputs=['a'],
-    outputs=['b'],
+    outputs=['b'], # Default value for axis: axis=1
 )
 
 shape = (5, 4, 3, 2)
-axis_default_value = 1
```
Comment (spandantiwari) [inline docs/Operators.md]: This was actually a vestige from my previous PR https://github.com/onnx/onnx/pull/448 . 
Removed as suggested by @bddppq.

File: onnx/backend/test/case/node/conv.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                                    [  72.,  117.,   84.]]])
+        expect(node_with_padding, inputs=[x, W], outputs=[y_with_padding],
+           name='test_conv_with_strides_padding')
+
+        # Convolution with strides=2 and no padding
+        node_without_padding = onnx.helper.make_node(
+            'Conv',
+            inputs=['x', 'W'],
+            outputs=['y'],
+            kernel_shape=[3, 3],
```
Comment (houseroad) [inline onnx/backend/test/case/node/conv.py]: I would suggest some cases where we have different width and height dimension. And also test different paddings on width and height dimension.

File: onnx/backend/test/case/node/conv.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+from ..base import Base
+from . import expect
+
+
+class Conv(Base):
+
+    @staticmethod
+    def export():
+
+        x = np.array([[[  0.,   1.,   2.,   3.,   4.], # (1, 5, 5) input tensor
```
Comment (houseroad) [inline onnx/backend/test/case/node/conv.py]: This should be 4-D tensor, please check the [spec](https://github.com/onnx/onnx/blob/master/docs/Operators.md#inputs-2---3)

File: onnx/backend/test/case/node/conv.py (hunk 2 above)
Comment (spandantiwari) [reply onnx/backend/test/case/node/conv.py]: I agree. I have modified the second test case export_conv_with_strides() to have input with different width and height. Also added a third test point in the same that has different padding for width and height dimensions, as suggested.

File: onnx/backend/test/case/node/conv.py (hunk 3 above)
Comment (spandantiwari) [reply onnx/backend/test/case/node/conv.py]: Fixed.

**Comment by spandantiwari** [comment]
@houseroad - I have updated the change as per your feedback. Could you please take a look?

**Comment by houseroad** [comment]
Just like what we did here: https://github.com/onnx/onnx/blob/master/onnx/backend/test/case/node/abs.py#L22

**Comment by spandantiwari** [comment]
@houseroad - OK, done.

**Pull request merged**